2018-09-08 12:23:52 INFO  [zs.com.ina.db.mysql.pool.DBPool] [branch=] [user=] Error While Creating Connection :=> com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
